lib/CPathfinder.h

```diff
@@ -21,6 +23,19 @@ struct TerrainTile;
 
 struct DLL_LINKAGE CGPathNode
 {
+	typedef EPathfindingLayer ELayer;
+
+	enum ENodeAction
```

Comment:
> 2) Padding - Could be already fixed by Warmonger in another branch

Yes, I started such branch for all the map structures. Of course it break savegame, so better get done with current branch first.